package/corrade/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the Corrade Arch Linux PKGBUILD maintained by xyproto and the MSYS2/MinGW PKGBUILD templates
+
+_realname=corrade
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2018.10
+pkgrel=1
+pkgdesc='C++11/C++14 multiplatform utility library'
+provides=('corrade')
```

Comment:
:thinking: hmmm. Not sure about this line. Any reason why is it here? Otherwise I would just remove it.

Reply:
Yeah, you're right. I'll remove it.